drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/data_warehouse_upload_job.rb

```diff
@@ -7,6 +7,7 @@
 # job = HmisExternalApis::AcHmis::DataWarehouseUploadJob.new
 module HmisExternalApis::AcHmis
   class DataWarehouseUploadJob < BaseJob
+    queue_as ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)
```

Comment:
Would it help if these were combined into one job? We put several on the queue https://github.com/greenriver/hmis-warehouse/blob/045cbf9398249621cf8cc0212ae7a9adda5a0b3c/drivers/hmis_external_apis/lib/tasks/export.rake#L9-L19

Reply:
Not sure if that makes any difference, the'll run simultaneously if you do what you have.  If you put them all in one job, they'd run sequentially.